### PR TITLE
Fixes #147

### LINKED ISSUE #147: NMS.MA.ACCESS.007, NMS.MK.ACCESS.007 - Whitelist
NMS shall be configurable at a state level allowing a whitelist of CUG SIM/MSISDN numbers to be uploaded through MDS via a csv file.

The whitelist can be on or off at a state level.  When whitelisting is on, the calling number should be compared to the uploaded whitelist.  If the number is found in the list the call shall proceed as normal.  If the number is not in the list the call should be disconnected.

### PR DESCRIPTION
**Pull request by larubbio**
Add the ability to turn whitelist on or off by state

### CODE REVIEW

File: api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java [hunk 1]
```diff
@@ -712,6 +722,7 @@ public void testGetUserNotInWhitelistedByState() throws IOException, Interrupted
                 ADMIN_USERNAME, ADMIN_PASSWORD));
     }
 
+    @Ignore // TODO: Renable once #119 is merged
     @Test
     public void testGetUserNotInWhitelistedByLanguageLocationCode() throws IOException, InterruptedException {
```
Comment (llavoie) [inline api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java]: Nitpick: the name of this test is not grammatically correct -- testGetUserNotInWhitelistByLanguageLocationCode?

Comment (larubbio) [inline api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java]: I updated this test and the other.  ok to merge?

**Comment by llavoie** [comment]
looks fine

File: api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java (hunk 1 above)
Comment (llavoie) [inline api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java]: Yep, as long as CI passes

-----Original Message-----

**Pull request merged**